### DISCUSSION TITLE
May I merge the extension version into this repository?

### DISCUSSION THREAD

**Discussion by nonnonstop** [ZhUyU1997/open-pose-editor#55] [Ideas]
@ZhUyU1997 
As you know, I am developing the webui extension version of this editor.
https://github.com/nonnonstop/sd-webui-3d-open-pose-editor
I think it would be better to merge this into your repository. What do you think?

Previously, they were managed in separate repositories because of many differences between extension and online version.
But now, with your help, the code is almost same and there is little advantage to separate repositories.
Therefore, I'm willing to integrate my repository with yours.

Thank you for everything.

**Comment by ZhUyU1997** [comment]
No problem. I will also invite you to the developers.

**Reply by nonnonstop** [reply]
Thank you. I will submit a pull request later.

**Comment by nonnonstop** [comment]
# Task list

- [x] Add build scripts
  - Merge from [sd-webui-3d-open-pose-editor](https://github.com/nonnonstop/sd-webui-3d-open-pose-editor).
- [ ] Add pre-built files (`pages` and `javascript`)
  - I want to distribute those files via GitHub Releases, but I don't know how to do. (Help wanted)
- [ ] Change the repository URL in the a1111 extension list
  - [stable-diffusion-webui-extensions](https://github.com/AUTOMATIC1111/stable-diffusion-webui-extensions)
- [ ] Change the repository URL for existing users
  - Display migration messages in the webui. Or, automatically change the git remote repository at webui startup.
- [ ] Archive the old repository (several months later)
